helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -49,6 +54,12 @@ import org.ossreviewtoolkit.helper.commands.VerifySourceArtifactCurationsCommand
 import org.ossreviewtoolkit.helper.commands.packageconfig.PackageConfigurationCommand
 import org.ossreviewtoolkit.helper.commands.repoconfig.RepositoryConfigurationCommand
 import org.ossreviewtoolkit.helper.common.ORTH_NAME
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.model.writeValue
+import org.ossreviewtoolkit.utils.formatSizeInMib
+import org.ossreviewtoolkit.utils.log
+import org.ossreviewtoolkit.utils.perf
```

Comment:
I believe it would make sense to move them into the Utils class.

Reply:
I wanted to have them in a similar place as in the `cli` to make it easier if both need updates in the future.

Reply:
Is it really harder to updated if they are in the Utils file?

Reply:
It feels more natural to me that the subcommands use function from `common/Utils` rather than from `Main`, because they don't need to know about main. I won't insist on it though.

Reply:
Not easier to implement, but easier to find. However, I've implemented the change.